pkg/detectors/github/v1/github_old.go

```diff
@@ -19,9 +19,10 @@ type Scanner struct{ detectors.EndpointSetter }
 var _ detectors.Detector = (*Scanner)(nil)
 var _ detectors.Versioner = (*Scanner)(nil)
 var _ detectors.EndpointCustomizer = (*Scanner)(nil)
+var _ detectors.CloudProvider = (*Scanner)(nil)
 
-func (Scanner) Version() int            { return 1 }
-func (Scanner) DefaultEndpoint() string { return "https://api.github.com" }
+func (Scanner) Version() int          { return 1 }
+func (Scanner) CloudEndpoint() string { return "https://api.github.com" }
```

Comment:
**Question**: Is there a reason we're changing `DefaultEndpoint` to `CloudEndpoint`? I feel like `DefaultEndpoint` is much easier to wrap my head around.

Reply:
As per @mcastorina `EndpointCustomizer` was created for detectors that have cloud and on-prem endpoints. So we renamed it accordingly.

Reply:
`DefaultEndpoint` felt ambiguous and was really only defining a default *static* endpoint. Would `DefaultCloudEndpoint()` be a better name?

Reply:
`CloudEndpoint` is fine, thanks for the explanation